modules/plugin-search.php

```diff
@@ -15,6 +16,7 @@
 	Jetpack::is_active() &&
 	/** This filter is documented in _inc/lib/admin-pages/class.jetpack-react-page.php */
 	apply_filters( 'jetpack_show_promotions', true ) &&
+	! Constants::is_true( 'DISALLOW_FILE_MODS' ) &&
```

Comment:
I'd suggest an inline comment noting why we're checking this.

Reply:
👍 Done in 983d9e81c4baaa864292b3eb68c0d497a81f1eeb